source/help/apps/desktop-guide.rst

```diff
@@ -54,57 +54,40 @@ To remove a server from your desktop app environment:
-
-    This setting is enabled by default.
+    Highlight misspelled words in your messages based on your system language configuration. This setting is enabled by default. You must restart the app to change this setting. 
 
 Show red badge on taskbar icon to indicate unread messages (Windows, Mac only)
-    When enabled, a red badge is shown on the taskbar icon for unread messages with a number count indicating unread mentions or direct messages. If disabled, a red badge is only shown for unread mentions (with a number count).
-
-    This setting is enabled by default.
+    When enabled, a red badge is shown on the taskbar icon for unread messages with a number count indicating unread mentions or direct messages. If disabled, a red badge is only shown for unread mentions (with a number count). This setting is enabled by default.
 
 Flash taskbar icon when a new message is received (Windows, Linux only)
-    Configure whether the taskbar icon flashes for a few messages when a new message is received on any of your active teams and servers.
-
-    This setting is enabled by default.
+    Configure whether the taskbar icon flashes for a few messages when a new message is received on any of your active teams and servers. This setting is enabled by default.
     
 Bounce the Dock icon when receiving a notification (Mac only)
-    When enabled, the Dock icon will either bounce once or bounce until the user opens the app when receiving a notification.
-    
-    This setting is enabled by default.
+    When enabled, the Dock icon will either bounce once or bounce until the user opens the app when receiving a notification. This setting is enabled by default.
 
 Show Mattermost icon in the menu bar (Mac only)
-    When enabled, Mattermost icon is added to the Mac menu bar.
-
-    This setting is disabled by default.
+    When enabled, Mattermost icon is added to the Mac menu bar. This setting is disabled by default.
 
 Show icon in the notification area (Linux only)
-    When enabled, Mattermost icon is added to the Linux notification area.
-
-    This setting is enabled by default.
+    When enabled, Mattermost icon is added to the Linux notification area. This setting is enabled by default.
 
 Leave app running in notification center when application window is closed (Linux only)
-    When enabled, closing the application window will leave the Mattermost desktop app running in your notification center. This can be useful if you want to check for unread mentions while away from the app.
-
-    This setting is enabled by default.
+    When enabled, closing the application window will leave the Mattermost desktop app running in your notification center. This can be useful if you want to check for unread mentions while away from the app. This setting is enabled by default.
```

Comment:
```suggestion
    When enabled, closing the application window leaves the Mattermost Desktop App running in your notification center. This can be useful if you want to check for unread mentions while away from the app. This setting is enabled by default.
```